changes/en-us/2.0.0.md

```diff
@@ -17,6 +17,8 @@ The version is updated as follows:
 ### feature：
 - [[#5165](https://github.com/seata/seata/pull/5165)] optimize TCC structure, supporting API access. add integration layer module(seata-integration-tx-api) for transaction process definition and proxy enhancement.
 - [[#5352](https://github.com/seata/seata/pull/5352)] add jackson json parser and gson json parser for tcc business action context
+- [[#5377]( https://github.com/seata/seata/pull/5377)] make AbstractHttpExecutor.class  support http put
```

Comment:
You can modify the code and push to your remote repository and branch directly.

Reply:
just push the code to [zsp419:support_http_put](https://github.com/zsp419/seata/tree/support_http_put)